src/main/java/com/internet/shop/controller/user/DeleteUserController.java

```diff
@@ -26,7 +26,7 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
         userService.delete(userId);
         HttpSession session = req.getSession();
         if (userId.equals(session.getAttribute(LoginController.USER_ID))) {
```

Comment:
I think this block will not work in this place
this controller should be available only to the admin

if the admin will decide to remove himself, we will get an exception on line 25 (because admin does not have a shopping cart)

Reply:
This controller is available only to the admin
Added checking to avoid exception